R/crunch-page.R

```diff
@@ -33,51 +35,16 @@
 #'     )
 #' )
 #' }
-crunchPage <- function (...) {
-    fluidPage(
-        loadCrunchAssets(),
-        crunchAuthPlaceholder(),
-        ...
-    )
-}
+crunchPage <- function (...) fluidPage(...)
```

Comment:
using `crunchPage <- function (...) fluidPage(...)` instead of `crunchPage <- fluidPage` is to get around that problem we had where httptest had an old version of a testthat function kicking around until cran recompiled httptest's binaries with the new testthat around, yeah?

Reply:
No, though that would probably have been smart tool. It's because if (when) I did `crunchPage <- fluidPage`, I'd have to document all of the input parameters of `shiny::fluidPage` in order to pass `check`. So, strategic laziness.

Reply:
Also a good reason! And beneficial side-effect it _should_ also prevent that stale definition problem.